### DISCUSSION TITLE
dacia duster

### DISCUSSION THREAD

**Discussion by titus091994** [cedricp/ddt4all#1303] [General]
bonjour quelq un aurai t il la mise a jour 9.1.1 pour le gps duster 2023 merci de me dire ou je pourrai la telecharger ?

**Comment by AndyIreland** [comment]
here is the strange thing 9.1.1 is a medianav evo radio update - you would hardly have an old fashioned medianav fitted on a modern 2023 car surely . if it is definitely a medianav fitted then 9.1.3 is the latest firmware for that :
https://www.gps-rlink.com/medianav/update-software-medianav/

**Comment by titus091994** [comment]
bonne question je ne sais pas quelle version j ai la actuellement j ai la
version 9.0.12.2
Cordialement

Eric Esparrell

Tel 06 88 50 94 87


Le sam. 30 nov. 2024 à 14:07, Andy Robus ***@***.***> a
écrit :
